src/ploomber/telemetry/telemetry.py

```diff
@@ -62,6 +63,50 @@ def is_online(host='http://ploomber.io'):
         return False
 
 
+# Will output if the code is within a container
+def is_docker():
+    path = '/proc/self/cgroup'
+    return (os.path.exists('/.dockerenv')
```

Comment:
not a big deal but consider using pathlib.Path instead of os.path, it has a cleaner API and it's what we use in other modules